sssd_test_framework/utils/pam.py

```diff
@@ -0,0 +1,235 @@
+""""PAM Tools."""
+
+from __future__ import annotations
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.utils.fs import LinuxFileSystem
+
+__all__ = [
+    "PAMUtils",
+    "PAMAccess",
+    "PAMFaillock",
+]
+
+
+class PAMUtils(MultihostUtility[MultihostHost]):
+    """
+    Management of PAM modules
+    """
+
+    def __init__(self, host: MultihostHost, fs: LinuxFileSystem) -> None:
+        """
+        :param host: Remote host instance
+        :type host: MultihostHost
+        """
+        super().__init__(host)
+
+        self.file = None
+        self.fs: LinuxFileSystem = fs
+
+    def access(self, file: str | None = None) -> PAMAccess:
+        """
+        :param file: PAM Access file name.
+        :type file: str, optional
+        :return: PAM Access object
+        :rtype: PAMAccess
+        """
+        if file is None:
+            return PAMAccess(self)
+
+        return PAMAccess(self, file)
+
+    def faillock(self, user: str | None = None, file: str | None = None) -> PAMFaillock:
```

Comment:
Same here, keep the defaults the same as PAMFaillock constructor. Then you can use just `return PAMFaillock(self, user, file)`.